docs-ui/components/text.stories.js

```diff
@@ -4,7 +4,7 @@ import {withInfo} from '@storybook/addon-info';
 import Text from 'app/components/text';
 
 export default {
-  title: 'Style/Text',
+  title: 'Core/Style/Text',
```

Comment:
We should refactor our type styles and then refactor this documentation because it’s extremely misleading — we shouldn’t be using this component at all.